.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,57 @@
+name: Ensure the operator builds
+on:
+  push:
+    branches:
+      - 'master'
+      - 'v[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+-[0-9a-zA-Z]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[0-9a-zA-Z]+'
+  pull_request:
+    branches:
+      - 'master'
+      - 'v[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+-[0-9a-zA-Z]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-[0-9a-zA-Z]+'
+
+jobs:
+
+  build-operator:
+    runs-on: ubuntu-latest
+
+    env:
+      ARGOCD_OPERATOR_IMAGE_BUILDER: docker
+      GO111MODULE: on
```

Comment:
`GO111MODULE` should not be needed. Do you run into a problem without it?

Reply:
OK, I've removed it.